### PR TITLE
Jarvis, add buckshot to the blackmarket.

### PR DESCRIPTION
**Pull request by Ghommie**
## About The Pull Request
immagine

~discord light-theme big L.~

By the by, these are slightly nerfed buckshots, create big puffs of powder smoke when fired, damage your gun, and MIGHT blow you up in the same style of using a detective revolver loaded with .357 if you fire too many.

Also, the integrity of the firearm now counts towards the damage of bullets it's fired from.

## Why It's Good For The Game
The blackmarket is a place where you can find illegally illegal, evil items, along with other trinkets. I thought it'd be a nice place to reintroduce buckshot with a little downgrading twist after it has been nerfed (it used to do 60 damage without falloff) AFTER it was removed from the lathes and the station.

## Changelog

:cl:
add: Buckshot is back on the menu, on the blackmarket.
balance: the integrity of firearms now counts toward projectile damage. A gun that's on the very verge of breaking down will deal half as much damage.
/:cl:

### CODE REVIEW

File: code/modules/cargo/markets/market_items/weapons.dm [hunk 1]
```diff
@@ -24,6 +24,16 @@
 	stock_max = 4
 	availability_prob = 40
 
+/datum/market_item/weapon/buckshot
+	name = "Box of Buckshot Shells"
+	desc = "It wasn't easy since buckshot has been made illegal all over the sector of space, but \
```
Comment (nikothedude) [inline code/modules/cargo/markets/market_items/weapons.dm:29]: all over this sector of space*

this, not the

**Comment by Eronymun** [comment]
Jarvis, you've done good.

**Comment by CCC23** [comment]
Finally BookShot has been added to the funny market

**Comment by Ghommie** [comment]
Provide a reason to why you don't want this, optimumtact.

**Comment by CCC23** [comment]
orange man said buckshot bad

**Comment by optimumtact** [comment]
Don't remove a label i've added, I was called away before I could finish my comment.

I wanted to ask what the reasoning was behind removing the wound bonus, since the comment indicates it's some callback to old code before wounds, but that makes no sense, if it's just for balance then the comment should just say it is explicitly to make it less powerful than the buckshot you can get normally otherwise someone will just inadvertently undo it later.

**Comment by carlarctg** [comment]
Why are you adding a rng chance to blow up, that shit always sucks. Don't stack 30 downsides on top of eachother.

**Comment by CCC23** [comment]
> Why are you adding a rng chance to blow up, that shit always sucks. Don't stack 30 downsides on top of eachother.

you are buying bubbas hot load off the black market and expecting the ammo to NOT explode in your face?

**Comment by carlarctg** [comment]
> > Why are you adding a rng chance to blow up, that shit always sucks. Don't stack 30 downsides on top of eachother.
> 
> you are buying bubbas hot load off the black market and expecting the ammo to NOT explode in your face?

Hurr durr. No. It's not funny ingame. It's just infuriating. No one's going out of their way to buy black market ammo if it can also fuck them over and kill them, it's an insult to the player. Make it corrode guns if you have to - that's what black powder does to modern firearms.

**Comment by Ghommie** [comment]
drafting until I rework it so it corrodes the gun instead.

**Comment by optimumtact** [comment]
maybe it should corrode carlactg's brain

**Comment by Loafin34** [comment]
maybe instead of having it corrode the shell primer is eroded so shots have a 1/5 chance of being spent already (you can't tell by examining though)

**Comment by CCC23** [comment]
could do a variable damage thing some loaded hot so about 1.5x damage some being duds and doing like 1 damage per pellet

**Pull request merged**